website/src/remark/render-code-blocks.mjs

```diff
@@ -171,12 +171,18 @@ export default function plugin() {
             const skipErr = meta && meta.match(skipRegex) !== null;
             const editableRegex = new RegExp(/(always-editable)/g);
             const alwaysEditable = meta && meta.match(editableRegex) !== null;
+            const lineNumRegex = new RegExp(/(showLineNumbers)/g);
 
             for (const langConfig of languageConfig.languages) {
 
                 const label = langConfig.label;
                 const highlight = langConfig.highlight;
 
+                // line numbers can be shown for all blocks through `language.config.js`,
+                // or for a specific block through `showLineNumbers`
+                // e.g. ```z3 showLineNumbers
+                const showLineNumbers = langConfig.showLineNumbers || meta && meta.match(lineNumRegex) !== null;
```

Comment:
use `?.` operator to call a potentially nully instance method

    meta && meta.match(lineNumRegex)

to

    meta?.match(lineNumRegex)

Reply:
resolved in #98